src/test/rustdoc-ui/scrape-examples-ice.rs

```diff
@@ -0,0 +1,4 @@
+// compile-flags: -Z unstable-options --scrape-examples-output-path t.calls --scrape-examples-target-crate foobar
```

Comment:
I think you can just remove `output-path` here.

Reply:
Don't think so, it will error if either of the two options are unset. Perhaps it should point to build/tmp/?

Reply:
> it will error if either of the two options are unset.

well that's odd ... @willcrichton is there a reason you made that a hard error instead of giving it a default value?

anyway, in the meantime, try `{{build-base}}/t.calls`: https://rustc-dev-guide.rust-lang.org/tests/adding.html#input-normalization

Reply:
There isn't an obvious default value IMO. Eg doing `{crate_name}.calls` isn't desirable due to the many conflicting crate names in a workspace.

Reply:
We could do `{crate_name}-{crate_type}` maybe. I don't think it has to be *amazing*, just some reasonable default to match `--output-format` json.

Reply:
Any tool that cares about conflicts should be using `output-path` like you suggested.